minigui/README.md

```diff
@@ -72,15 +38,18 @@
 
 **C++ Engine**
 
-
 1. By default, Minigui will use the Python Minigo engine. You can use the C++
    engine by first compiling it (see the
    [README](https://github.com/tensorflow/minigo/tree/master/cc/README.md)),
    then passing `--engine=cc` when starting `minigui/serve.py`. You will need
    to pass a frozen GraphDef proto as the `--model` command line argument
    instead of the saved parameter data that the Python backend requires.
 
+   **Note:** Compiling tensorflow from scratch can take 2+ hours. So you might
```

Comment:
depends strongly on number of cores in the machine.  It's about 15-30 minutes on my corpstation